src/main/java/com/epam/reportportal/cucumber/Utils.java

```diff
@@ -299,6 +305,25 @@ public static TestCaseIdEntry getTestCaseId(TestStep testStep, String codeRef) {
 		}
 	}
 
+	static List<ParameterResource> getParameters(List<cucumber.api.Argument> arguments, String text) {
+		List<ParameterResource> parameters = Lists.newArrayList();
+		Optional<String> parameterName = Optional.empty();
+		Matcher matcher = Pattern.compile(PARAMETER_REGEX).matcher(text);
+		if (matcher.find()) {
+			parameterName = Optional.of(text.substring(matcher.start() + 1, matcher.end() - 1));
+		}
+		if (!arguments.isEmpty() && parameterName.isPresent()) {
```

Comment:
The same as for each cucumber agent your code does not take into account many parameters.

Reply:
fixed